framework/doc/content/source/ics/FunctionIC.md

```diff
@@ -1,33 +1,14 @@
 # FunctionIC
 
-!alert! construction title=Undocumented Class
-The FunctionIC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
-```markdown
-# FunctionIC
-
 !syntax description /ICs/FunctionIC
 
-## Overview
+Sets an initial condition via a [Function](syntax/Functions/index.md) described by parameter [!param](/ICs/FunctionIC/function). It can be restricted to particular blocks and boundaries using the [!param](/ICs/FunctionIC/block) and !param](/ICs/FunctionIC/boundary) parameters, respectively.
```

Comment:
```suggestion
Sets an initial condition via a [Function](syntax/Functions/index.md) described by parameter [!param](/ICs/FunctionIC/function). It can be restricted to particular blocks and boundaries using the [!param](/ICs/FunctionIC/block) and [!param](/ICs/FunctionIC/boundary) parameters, respectively.
```